layouts/partials/footer.html

```diff
@@ -96,3 +96,34 @@
 
 </script>
 {{- end }}
+
+{{- if (not .Site.Params.disableCodeCopy) }}
+<script>
+    document.querySelectorAll('pre > code').forEach((codeblock) => {
```

Comment:
Maybe we can add the following here and place `copyingDone(copybutton);` at `$2` and `$3`

```js
    function copyingDone(copybutton) {
        copybutton.innerText = 'copied!';
        setTimeout(function () {
            copybutton.innerText = 'copy';
        }, 2000);
    }
```